app/src/App.tsx

```diff
@@ -37,14 +38,16 @@ const App = () => {
     <ThemeProvider theme={theme}>
       <BrowserRouter>
         <Routes>
+          {/* Normal pages */}
```

Comment:
What is this comment for? (I would remove it since I don't think it adds much value.)

Reply:
Alex says this has proper context with another comment that will be added in a future PR.